lsstseries/timeseries.py

```diff
@@ -0,0 +1,41 @@
+import pandas as pd
+import numpy as np
+
+class timeseries():
+    def __init__(self,data=None):
+        self.data = data
```

Comment:
With the fullest understanding this is only a prototype, so feel free to ignore; This allows the user to pass in the data that is not sorted, i.e., one can pass data that is not ready to be used in any way.